onnxruntime/contrib_ops/cuda/bert/flash_attention/flash_fwd_launch_template.h

```diff
@@ -122,6 +125,11 @@ void run_mha_fwd_splitkv_dispatch(Flash_fwd_params& params, cudaStream_t stream)
     constexpr int kBlockN = Headdim <= 64 ? 256 : (Headdim <= 128 ? 128 : 64);
     run_flash_splitkv_fwd<Flash_fwd_kernel_traits<Headdim, kBlockM, kBlockN, 4, false, false, T>>(params, stream);
   }
+#else
+  (void)params;
+  (void)stream;
+  throw std::runtime_error("FlashAttention is only implemented for SM>=80");
```

Comment:
Use ORT_THROW instead?

Reply:
It is non-ORT related code. I don't want to introduce ORT files to here.